sql/moz-fx-data-shared-prod/monitoring_derived/airflow_dag_run_v1/query.sql

```diff
@@ -7,6 +7,6 @@ SELECT
   start_date,
   end_date,
 FROM
-  `moz-fx-data-bq-fivetran.airflow_metadata_airflow_db.dag_run`
+  `moz-fx-data-bq-fivetran.telemetry_airflow_metadata_public.dag_run`
```

Comment:
I don't think they kept any historical data in the migration, right?  So just using the new database in all these ETLs means we lose all history prior to the migration.  Should we include both the current and historical data instead?

Reply:
hmmm good point ... it's mainly for an airflow monitoring dashboard so not sure how much value old stuff has ... but it couldn't hurt either

Reply:
ok, now including data pre-migration for `dag_run`, `task_fail`, and `task_reschedule` tables.